test/public-api/package.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
We might also want to add a test with browser mode active to see if Metada gets passed correctly there as well... ?